src/components/Footer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const Footer: React.FC = () => {
+    return (
+        <>
+            <div className='mt-10 px-20 fixed bottom-0 w-full border-t py-3 bg-white'>
+                <div className='flex justify-between items-center'>
+                    <div>
+                        <ul className='flex items-center gap-3'>
+                            <li className=' decoration-0 no-underline'>About</li>
```

Comment:
Remove extra space  in class name.
The styles look repetitive, probably move it to parent ?

Reply:
fixed